cli/utils.js

```diff
@@ -184,19 +176,18 @@ const getCommand = (cmd, args, opts) => {
     return `${cmd}${cmdArgs ? ` ${cmdArgs}` : ''}`;
 };
 
-const getCommandOptions = (pkg, argv) => {
+const addKebabCase = (options = {}) => {
+    const kebabCase = Object.keys(options).reduce((acc, key) => {
+        acc[_.kebabCase(key)] = options[key];
+        return acc;
+    }, {});
+    return { ...kebabCase, ...options };
+};
+
+const getCommandOptions = (pkg, argv = []) => {
```

Comment:
add or update tests plz

Reply:
Added addKebabCase test, getCommandOptions is already updated.